.github/workflows/new_issues.yml

```diff
@@ -16,4 +16,4 @@ jobs:
         with:
           project: Openverse
           column: Backlog
-          repo-token: ${{ secrets.ACCESS_TOKEN }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
`GITHUB_TOKEN` [might not have access](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) to projects outside the repo.